### ISSUE TITLE
Internal error in Immediate Window when trying to create an already existing variable

### ISSUE THREAD

**Issue by mattscheffer** [dotnet/roslyn#22599]
**Version Used**: 
D15svc 27004.2002
**Steps to Reproduce**:
1. Create a new C# console app.
2. Add some code to create a variable and add a breakpoint.
```
string s = "foo";
System.Diagnostics.Debugger.Break();
```
3.  Start Debugging
4. Open immediate window and try to create the existing variable s. Do something like:
```
string s = "asdf";
````

**Expected Behavior**:
It should throw an error explaining that the variable already exists.
**Actual Behavior**:
It sends an error saying "Internal error in the C# compiler"

No other ill effects appear to occur. Immediate window still works properly after the error.

**Comment by tmat** [comment]
@ivanbasov Could you investigate? This seems like a very basic scenario. 
/cc @jinujoseph

**Comment by ivanbasov** [comment]
I believe that the root cause is the same as at https://github.com/dotnet/roslyn/issues/22474. Not sure if it meets the bar for 15.5 but it would be nice to schedule it at least for 15.6.

**Comment by ivanbasov** [comment]
@jinujoseph, I could not find an easy way to fix yet. I'm not sure that this meets 15.5 bar: it seems that this is a bad diagnostic message. I suggest to punt it to 15.6 and consider moving it back if we will find an the easy fix.

**Comment by ivanbasov** [comment]
Resolved with https://github.com/dotnet/roslyn/pull/24198
